src/main/java/sirius/kernel/Sirius.java

```diff
@@ -67,9 +67,10 @@
     protected static final Log LOG = Log.get("sirius");
 
     private static final String DEBUG_LOGGER_NAME = "debug";
+    private static final String CONFIG_INSTANCE = "instance";
 
     /**
-     * This debug logger will be logging all messages when {@link sirius.kernel.Sirius#isDev()} is true. Otherwise,
+     * This debug logger will be logging all messages when {@link Sirius#isProd()} is false. Otherwise,
```

Comment:
Doesnt this a lot more is logged in test/staging?

Reply:
Not sure if this is necessarily a bad thing. Compared to prod there is much less traffic, so the logs shouldnt be too much to handle, even with debug logging